Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/configuration/jmx/AbstractPDDeferredRequestWebServicesMXBean.java

```diff
@@ -73,22 +73,22 @@
     /**
      * The Constant Standard_OutboundOrch_PDDeferredRequest_BEAN_NAME.
      */
-    private static final String StdOutbound_PD_Bean_Name = "stdPDReqOutbound";
+    private static final String StdOutbound_PD_Bean_Name_Req = "stdPDReqOutbound";
```

Comment:
Strange how some of the constant strings follow convention and are all caps and some don't.  I'd suggest updating to be consistent.

Reply:
Fixed